runtime-assets.sln

```diff
@@ -144,17 +144,20 @@ Global
 		{BB874A4E-3388-44EB-B4DC-40C0F38572C8}.Release|x64.Build.0 = Release|Any CPU
 		{BB874A4E-3388-44EB-B4DC-40C0F38572C8}.Release|x86.ActiveCfg = Release|Any CPU
 		{BB874A4E-3388-44EB-B4DC-40C0F38572C8}.Release|x86.Build.0 = Release|Any CPU
-		{3A28B303-88E0-43B2-BFDF-9D388CC283FB}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{3A28B303-88E0-43B2-BFDF-9D388CC283FB}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{3A28B303-88E0-43B2-BFDF-9D388CC283FB}.Debug|x64.ActiveCfg = Debug|Any CPU
-		{3A28B303-88E0-43B2-BFDF-9D388CC283FB}.Debug|x64.Build.0 = Debug|Any CPU
-		{3A28B303-88E0-43B2-BFDF-9D388CC283FB}.Debug|x86.ActiveCfg = Debug|Any CPU
-		{3A28B303-88E0-43B2-BFDF-9D388CC283FB}.Debug|x86.Build.0 = Debug|Any CPU
-		{3A28B303-88E0-43B2-BFDF-9D388CC283FB}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{3A28B303-88E0-43B2-BFDF-9D388CC283FB}.Release|Any CPU.Build.0 = Release|Any CPU
-		{3A28B303-88E0-43B2-BFDF-9D388CC283FB}.Release|x64.ActiveCfg = Release|Any CPU
-		{3A28B303-88E0-43B2-BFDF-9D388CC283FB}.Release|x64.Build.0 = Release|Any CPU
-		{3A28B303-88E0-43B2-BFDF-9D388CC283FB}.Release|x86.ActiveCfg = Release|Any CPU
-		{3A28B303-88E0-43B2-BFDF-9D388CC283FB}.Release|x86.Build.0 = Release|Any CPU
+		{B1A890B0-F851-45A0-95D4-FA739061EC36}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{B1A890B0-F851-45A0-95D4-FA739061EC36}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{B1A890B0-F851-45A0-95D4-FA739061EC36}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{B1A890B0-F851-45A0-95D4-FA739061EC36}.Debug|x64.Build.0 = Debug|Any CPU
+		{B1A890B0-F851-45A0-95D4-FA739061EC36}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{B1A890B0-F851-45A0-95D4-FA739061EC36}.Debug|x86.Build.0 = Debug|Any CPU
+		{B1A890B0-F851-45A0-95D4-FA739061EC36}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{B1A890B0-F851-45A0-95D4-FA739061EC36}.Release|Any CPU.Build.0 = Release|Any CPU
+		{B1A890B0-F851-45A0-95D4-FA739061EC36}.Release|x64.ActiveCfg = Release|Any CPU
+		{B1A890B0-F851-45A0-95D4-FA739061EC36}.Release|x64.Build.0 = Release|Any CPU
+		{B1A890B0-F851-45A0-95D4-FA739061EC36}.Release|x86.ActiveCfg = Release|Any CPU
+		{B1A890B0-F851-45A0-95D4-FA739061EC36}.Release|x86.Build.0 = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(NestedProjects) = preSolution
+		{B1A890B0-F851-45A0-95D4-FA739061EC36} = {3C7C9597-003A-4216-A7BA-8215AAD5F81B}
```

Comment:
looks like this causes some build issue on CI?

```
runtime-assets.sln(2,1): error MSB4025: (NETCORE_ENGINEERING_TELEMETRY=Restore) The project file could not be loaded. Data at the root level is invalid. Line 2, position 1.
```

It mentions line 2 but you have no change there and this looks like the only relevant change.

Reply:
Yea I added the `System.Runtime.TimeZoneData.csproj` without `--in-root`. Shouldn't be a problem now